Fortran90/data_types/meso_approx.f90

```diff
@@ -196,7 +244,7 @@ subroutine approx_initialisation(appr)
      do i=1,6
       appr%hamilt%corr%intmap(i)=1
      end do
-     do i=7,13
+     do i=7,12
       appr%hamilt%corr%intmap(i)=2
      end do
      appr%hamilt%corr%value(1)=0.1d0
```

Comment:
Make a not in the code (or add an issue on Github), that this is a guess value and might need to be adjusted in the future.